### PR TITLE
[RA-1 Ch01] Language Fixes

### PR DESCRIPTION
**Pull request by pgoyal01**
1. Change Compliant to Conformant (Issue #988)
2. Changed language of 1.1 and 1.3

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter01.md [hunk 1]
```diff
@@ -33,9 +33,8 @@ Several NFV use cases are documented in OpenStack. For more examples and details
 
 <a name="1.3"></a>
 ## 1.3 Terminology
-Unless specific OpenStack terms are used, the terminology can be found [CNTT Reference Model Terminology](../../../ref_model/chapters/glossary.md#1.1).  
+The general terminology, except for this Reference ARchitecture specific terms, can be found [CNTT Reference Model Terminology](../../../ref_model/chapters/glossary.md#1.1); this Reference Architecture terms will be defined where used.
```
Comment (iangardner22) [inline doc/ref_arch/openstack/chapters/chapter01.md]: This currently doesn't read very clearly.
I assume you are saying
"General terminology definitions can be found [CNTT Reference Model] and specific terms relating to this reference architecture are to be found [here]" right?

File: doc/ref_arch/openstack/chapters/chapter01.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ This Reference Architectures is focussed on OpenStack as the VIM chosen based on
 
 <a name="1.1.1"></a>
 ## 1.1.1. Vision
-The ultimate goal is that the OpenStack-based CNTT Reference Architecture will host NFV workloads, primarily VNFs, of interest to the Common NFVI Telco community.  Initially the OpenStack-based Reference Architecture would be used for testing and validating VNFs that can then be deployed in compatible operator clouds.   The Reference Architecture document can be used by operators to deploy compliant production infrastructure.  
+The OpenStack-based CNTT Reference Architecture will host NFV workloads, primarily VNFs, of interest to the CNTT community.  The Reference Architecture document can be used by operators to deploy CNTT conformant infrastructure.
```
Comment (iangardner22) [inline doc/ref_arch/openstack/chapters/chapter01.md]: Should we be avoiding the use of the temporary term "CNTT"?

File: doc/ref_arch/openstack/chapters/chapter01.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter01.md]: @iangardner22 Changed - Thanks

File: doc/ref_arch/openstack/chapters/chapter01.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter01.md]: @iangardner22 CNTT is used throughout the documents -- RM, RA-1, ... This should be a TSC decision and if required to change should be made across all documents. Do you agree?

File: doc/ref_arch/openstack/README.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ This is Openstack based Reference Architecture
 
 
 ## Table of Contents
-* [Abbreviations](abbreviations.md)
+* [Terminology](chapter/terminology.md)
```
Comment (deebhatia) [inline doc/ref_arch/openstack/README.md]: The link doesn't work.

File: doc/ref_arch/openstack/chapters/chapter01.md [hunk 4]
```diff
@@ -33,9 +33,8 @@ Several NFV use cases are documented in OpenStack. For more examples and details
 
 <a name="1.3"></a>
 ## 1.3 Terminology
-Unless specific OpenStack terms are used, the terminology can be found [CNTT Reference Model Terminology](../../../ref_model/chapters/glossary.md#1.1).  
 
-Add add any OpenStack specific terms here.
+General terminology definitions can be found [CNTT Reference Model](../../../ref_model/chapters/glossary.md#1.1) and specific terms relating to this reference architecture are to be found [RA-1 Terminology](./terminology.md).
```
Comment (deebhatia) [inline doc/ref_arch/openstack/chapters/chapter01.md]: Terminology link doesn't work.

File: doc/ref_arch/openstack/README.md (hunk 3 above)
Comment (deebhatia) [reply doc/ref_arch/openstack/README.md]: Okay I saw that you have added a pending statement related to terminology chapter `Still need to create the chapter and content`. Will it be part of this pull request or a separate one?

File: doc/ref_arch/openstack/chapters/chapter01.md (hunk 2 above)
Comment (iangardner22) [reply doc/ref_arch/openstack/chapters/chapter01.md]: Yes agree entirely!

File: doc/ref_arch/openstack/README.md (hunk 3 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/README.md]: @deebhatia The Terminology chapter will be created as a separate PR.

**Comment by pgoyal01** [comment]
@rabiabdel Can you approve as code owner so we can merge and close this PR #1298 . Thanks

**Comment by rabiabdel** [comment]
48 hour since last addressed feedback. approving.

**Pull request merged**